app/admin/logs/api_logs.rb

```diff
@@ -16,7 +16,7 @@
 
   controller do
     def scoped_collection
-      super.select('id, created_at, status, method, path, db_duration, page_duration')
+      super.select('id, created_at, status, method, path, db_duration, page_duration, remote_ip')
```

Comment:
do we need to select `meta`?

Reply:
no, we do not need to select the meta attribute in the index page, I think showing meta in the show page will be enough.

Reply:
why then selecting `remote_ip` is needed?